crypto/des/cfb64ede.c

```diff
@@ -7,8 +7,8 @@
  * https://www.openssl.org/source/license.html
  */
 
-#include "des_locl.h"
 #include "e_os.h"
+#include "des_locl.h"
```

Comment:
One wonder same question [as with blake2] here. This is an algorithm implementation that can be done in pure C, there shouldn't be OS dependencies.

One can probably argue that this, removal of redundant e_os.h inclusions, should be matter for separate request...